swagger/v1/component_schemas/MultipleParticipantDeclarationsCsvResponse.yml

```diff
@@ -0,0 +1,14 @@
+description: "A list of participant declarations in the Comma Separated Value (CSV) format"
+type: string
+format: base64
```

Comment:
Is it?

Reply:
No it ain't.
I need to purge that from a few other places too...